src/objdump.main.v3

```diff
@@ -59,5 +62,88 @@ def parseAndDump(engine: Engine, path: string) -> int {
 		if (f.imp == null) OUT.put1(" { %d bytes }", f.orig_bytecode.length);
 		OUT.outln();
 	}
+	for (i < module.heaptypes.length) {
+		var h = module.heaptypes[i];
+		OUT.put1("heaptypes[%d] = ", i);
+		h.render(OUT);
+		var fh = h.getFirstSuperType();
+		if (fh != null) fh.render(OUT);
+		OUT.outln();
+	}
+	for (i < module.abstypes.length) {
+		var a = module.abstypes[i];
+		OUT.put1("abstypes[%d] = ", i);
+		if (a.isAssignableTo(ValueType.BOTTOM)) {
+			OUT.put1("%q ", ValueType.BOTTOM.render);
+		}
+		if (a.isAssignableTo(ValueType.I32)) {
+			OUT.put1("%q ", ValueType.I32.render);
+		}
+		if (a.isAssignableTo(ValueType.I64)) {
+			OUT.put1("%q ", ValueType.I64.render);
+		}
+		if (a.isAssignableTo(ValueType.F32)) {
+			OUT.put1("%q ", ValueType.F32.render);
+		}
+		if (a.isAssignableTo(ValueType.F64)) {
+			OUT.put1("%q ", ValueType.F64.render);
+		}
+		if (a.isAssignableTo(ValueType.V128)) {
+			OUT.put1("%q ", ValueType.V128.render);
+		}
+		OUT.outln();
+	}
+	for (i < module.tables.length) {
+		var t = module.tables[i];
+		OUT.put1("tables[%d] default elem = ", i);
+		match (t.default_elem) {
```

Comment:
Here, you should be able to use `t.name`, which is a string.

Reply:
It gives me error:
```
[src/objdump.main.v3 @ 99:28] UnresolvedMember: expression of type TableDecl has no such member "name"
		OUT.puts(t.name);
```
Also, do you mean I should replace the entire match block with one line of t.name?